kiwi/system/uri.py

```diff
@@ -179,15 +179,16 @@ def is_remote(self):
             raise KiwiUriStyleUnknown(
                 'URI scheme not detected %s' % self.uri
             )
-        if uri.scheme in self.remote_uri_types:
+        if uri.scheme == 'obs' and Defaults.is_buildservice_worker():
+            return False
```

Comment:
This change is applied always, inside and outside of the buildservice, I am not sure it will behave as we want in is_public method, as any obs URI in the buildserive will not be considered as a plublic address. That's why I initially thought to include a `check_build_environment` parameter in that method. 

In any case this is a quite specific case we can verify and polish after applying this patch so, so current builds turn back to normality.